Description: Don't catch crashes so apport can do its job

Index: gdm-2.30.2/daemon/factory-slave-main.c
===================================================================
--- gdm-2.30.2.orig/daemon/factory-slave-main.c	2010-03-18 08:27:16.000000000 +1100
+++ gdm-2.30.2/daemon/factory-slave-main.c	2010-06-22 13:10:57.736655391 +1000
@@ -229,7 +229,7 @@
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGFPE, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGHUP, signal_cb, NULL);
-        gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);
+        /*gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);*/
         gdm_signal_handler_add (signal_handler, SIGABRT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR1, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR2, signal_cb, NULL);
Index: gdm-2.30.2/daemon/product-slave-main.c
===================================================================
--- gdm-2.30.2.orig/daemon/product-slave-main.c	2010-03-18 08:27:16.000000000 +1100
+++ gdm-2.30.2/daemon/product-slave-main.c	2010-06-22 13:10:57.736655391 +1000
@@ -233,7 +233,7 @@
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGFPE, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGHUP, signal_cb, NULL);
-        gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);
+        /*gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);*/
         gdm_signal_handler_add (signal_handler, SIGABRT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR1, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR2, signal_cb, NULL);
Index: gdm-2.30.2/daemon/session-worker-main.c
===================================================================
--- gdm-2.30.2.orig/daemon/session-worker-main.c	2010-03-18 08:27:16.000000000 +1100
+++ gdm-2.30.2/daemon/session-worker-main.c	2010-06-22 13:10:57.736655391 +1000
@@ -194,7 +194,7 @@
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGFPE, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGHUP, signal_cb, NULL);
-        gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);
+        /*gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);*/
         gdm_signal_handler_add (signal_handler, SIGABRT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR1, signal_cb, NULL);
 
Index: gdm-2.30.2/daemon/simple-slave-main.c
===================================================================
--- gdm-2.30.2.orig/daemon/simple-slave-main.c	2010-04-27 03:08:42.000000000 +1000
+++ gdm-2.30.2/daemon/simple-slave-main.c	2010-06-22 13:10:57.736655391 +1000
@@ -241,7 +241,7 @@
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGFPE, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGHUP, signal_cb, NULL);
-        gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);
+        /*gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);*/
         gdm_signal_handler_add (signal_handler, SIGABRT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR1, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR2, signal_cb, NULL);
Index: gdm-2.30.2/daemon/xdmcp-chooser-slave-main.c
===================================================================
--- gdm-2.30.2.orig/daemon/xdmcp-chooser-slave-main.c	2010-03-18 08:27:16.000000000 +1100
+++ gdm-2.30.2/daemon/xdmcp-chooser-slave-main.c	2010-06-22 13:10:57.736655391 +1000
@@ -235,7 +235,7 @@
         gdm_signal_handler_add (signal_handler, SIGBUS, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGFPE, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGHUP, signal_cb, NULL);
-        gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);
+        /*gdm_signal_handler_add (signal_handler, SIGSEGV, signal_cb, NULL);*/
         gdm_signal_handler_add (signal_handler, SIGABRT, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR1, signal_cb, NULL);
         gdm_signal_handler_add (signal_handler, SIGUSR2, signal_cb, NULL);
